src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataResultSetLatestIT.java

```diff
@@ -32,4 +35,27 @@ public void testGetObjectNotSupported() throws SQLException {
       }
     }
   }
+
+  /** Added in > 3.17.1 */
```

Comment:
3.17.1 is not relesased yet, you can set there `> 3.17.0`

Reply:
You're right. Corrected this.